morganey-mode.el

```diff
@@ -62,6 +62,8 @@
   (make-local-variable 'comment-column)
   (setq comment-column 0))
 
+(global-set-key (kbd "C-\") (lambda () (interactive) (insert "λ")))
```

Comment:
It should set up via major mode map like https://github.com/codingteam/pacmacs.el/blob/d813e9c62c2540fe619234824fc60e128c786442/pacmacs.el#L99